Excercise2/js/script_ex_js_2.js

```diff
@@ -0,0 +1,30 @@
+window.addEventListener("load", function() {
+    const selectLocation = document.querySelector(".select-location");
+    const listLocation = document.querySelectorAll(
+        ".list-location .list-group-item"
+    );
+    selectLocation.addEventListener("change", function(e) {
+        let selected = this.value;
+        // clear active class
+        listLocation.forEach((location) => {
+            location.className = "list-group-item";
+        });
+        for (let index = 0; index < listLocation.length; index++) {
+            if (selected === "even") {
+                if (index % 2 !== 0) {
+                    listLocation[index].className += " active";
+                }
+            } else if (selected === "odd") {
+                if (index % 2 == 0) {
+                    listLocation[index].className += " active";
+                }
+            } else if (selected === "Select location" || selected === "reset") {
```

Comment:
e đưa condition này lên sau line 12, thỏa điều kiện thì return luôn, chuyển forEach thành for bình thường => Đỡ chạy loop, cải thiện performance